DigitalLearningSolutions.Web/Controllers/FrameworksController/Signposting.cs

```diff
@@ -1,22 +1,79 @@
 ﻿using DigitalLearningSolutions.Web.ViewModels.Frameworks;
 using Microsoft.AspNetCore.Mvc;
+using System;
+using System.IO;
 using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using DigitalLearningSolutions.Data.ApiClients;
+using System.Net.Http;
+using Microsoft.Extensions.Configuration;
 
 namespace DigitalLearningSolutions.Web.Controllers.FrameworksController
 {
     public partial class FrameworksController
     {
+        private static IConfiguration SignpostingConfiguration { get; set; }
+
         [Route("/Frameworks/{frameworkId}/Competency/{frameworkCompetencyId}/CompetencyGroup/{frameworkCompetencyGroupId}/Signposting")]
-        public IActionResult EditCompetencyLearningResources(int frameworkId, int? frameworkCompetencyGroupId = null, int? frameworkCompetencyId = null)
+        public IActionResult EditCompetencyLearningResources(int frameworkId, int? frameworkCompetencyGroupId = null, int? frameworkCompetencyId = null, string title = "")
         {
             var model = PopulatedModel(frameworkId, frameworkCompetencyId, frameworkCompetencyId);
+            model.Title = title;
             return View("Developer/EditCompetencyLearningResources", model);
         }
 
         [Route("/Frameworks/{frameworkId}/Competency/{frameworkCompetencyId}/CompetencyGroup/{frameworkCompetencyGroupId}/AddResources")]
-        public IActionResult AddCompetencyLearningResources(int frameworkId, int? frameworkCompetencyGroupId = null, int? frameworkCompetencyId = null)
+        public IActionResult AddCompetencyLearningResources(int frameworkId, int? frameworkCompetencyGroupId = null, int? frameworkCompetencyId = null, string Title = "")
         {
             var model = PopulatedModel(frameworkId, frameworkCompetencyId, frameworkCompetencyId);
+            model.Title = Title;
+            return View("Developer/AddCompetencyLearningResources", model);
+        }
+
+        [HttpPost]
+        public IActionResult AddCompetencyLearningResourceSummary(CompetencyResourceSummaryViewModel model)
```

Comment:
This method needs a Route template that extends that of the previous page. Something like:
`[Route("/Frameworks/{frameworkId}/Competency/{frameworkCompetencyId}/CompetencyGroup/{frameworkCompetencyGroupId}/AddResource/{resourceRefId}")]`